src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -90,19 +90,20 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         // isError == true if an exception occurred above
 
         boolean omitArchived = true;
-        data.instructors = loadCourseInstructorMap(omitArchived);
-        List<InstructorAttributes> instructorList =
-                new ArrayList<InstructorAttributes>(data.instructors.values());
-        data.courses = loadCoursesList(instructorList);
+        HashMap<String, InstructorAttributes> instructors = loadCourseInstructorMap(omitArchived);
+        ArrayList<InstructorAttributes> instructorList = new ArrayList<InstructorAttributes>(instructors.values());
```

Comment:
Does these need to be created as a `HashMap` or an `ArrayList`?
Would the `Map` or `List` interfaces work as well?